### PR TITLE
Init milestone1

### PR DESCRIPTION
**Pull request by marcode95**
-app is initialized with create-react-app
-unnecessary files are deleted
-big.js and prop-types are added to package.json
-app runs correctly with npm start
-app is deployed on netlify and the live link is included at the README file

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light:

Hi @marcode95, great effort in this milestone :+1: The app is initialized with `create-react-app` :100: App runs correctly with the `npm start` command :smile: 
However, there are a couple of changes we need to make before proceeding. Kindly check the comments below:
- Kindly remove all unnecessary files or assets, such as the React svg Logo, etc
- Add the following packages to the package.json file; `big.js`, `prop-types`
- Ensure the app is deployed on Heroku, Netlify, Firebase, etc
- Kindly add a descriptive README file. Every professional project has a descriptive README file to help other developers or non-technical people appreciate the project
- Kindly improve the PR, you can list the project requirements here. For example;
```
In this milestone, I performed the following:
- an app is initialized with create-react-app (create-react-app and Delete any unnecessary file or asset screenshot)

- unnecessary files or assets, such as the React svg Logo, are deleted.(screenshot from pathwright)

- big.js added to package.json. (screenshot from pathwright)

- prop-types added to package.json for typechecking props (screenshot from Pathwright)

- app runs correctly with npm start.(screenshot from pathwright)

- app is deployed on Heroku(or other hosts like Netfly, Firebase, etc) and the live link is included at the README file
```

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @marcode95  👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: README.old.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
\ No newline at end of file
+<div align="center">
+  <h1><b>Calculator</b></h1>
+
+A calculator app built with React.
+<br>  
+  
+
+## Live Version
+https://flamboyant-clarke-bee74a.netlify.app/
```
Comment (mikenath223) [inline README.old.md]: - [ ] It seems you mistakenly changed the readme filename from `readme.md` to `readme.old.md`. Kindly change the filename to the proper form to enable GitHub to render it on the branch page.

**Review by afanimartin — APPROVED** [review]
## Approved 👍🏿 👍🏿 👍🏿 
Nice work making the requested change to the readme file. Merge and proceed tot he next milestone.


### Martin Afani
#### Slack: whiz25

**Pull request merged**